app/styles/app.scss

```diff
@@ -175,19 +169,6 @@ tr.disabled {
   height: 95% !important;
 }
 
-.vl-modal {
-  .vl-modal__backdrop {
-    z-index: 52;
-  }
-  tbody {
-    background-color:white;
-  }
-}
-
-.vl-modal-dialog__content {
-  padding: 2rem 0;
-}
-
 .max-width {
   text-overflow: ellipsis;
   max-width: 800px;
```

Comment:
Code issue - usage of px - Ygdir standard 9 - Use rems

Reply:
This code was not introduced by me, and subsequently I will not expand beyond the scope of the issue.